app/package.json

```diff
@@ -31,10 +36,12 @@
     "@mui/material": "^5.8.6",
     "@reduxjs/toolkit": "^1.8.3",
     "formik": "^2.2.9",
+    "node": "^20.8.0",
```

Comment:
Is [this](https://github.com/hackforla/HomeUniteUs/pull/638/files#r1401513094) limited by line 39 here?

Reply:
@mira-kine, 

I believe this should be deleted. `node` is the runtime environment, not a packaged dependency. Specifying `node` here actually downloads [this package](https://www.npmjs.com/package/node) from the [node-bin-gen](https://github.com/aredridel/node-bin-gen) repository. It does not specify a `node.js` version. It looks like this repo is designed to install the LTS version of node and ignore the system node installs.